app/routes.py

```diff
@@ -1,2 +1,48 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, description, flag):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.flag = flag
+
+planets = [ 
+Planet(0, "jupiter", "largest planet in solar system", False),
+Planet(1, "mercury", "smallest planet in solar system", False), 
+Planet(2, "mars", "most explored planet in solar system", True)
+]
+
+planet_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planet_bp.route("", methods=["GET"])
+def list_planets():
+    planet_list = []
+    for planet in planets:
+        planet_list.append(dict(
+        id = planet.id,
+        name = planet.name,
+        description = planet.description,
+        flag = planet.flag
+        ))
+    return jsonify(planet_list)
+
+@planet_bp.route("/<id>", methods=["GET"])
+def get_planet(id):
+    planet = validate_planet(id)
+    return jsonify(dict(
+        id = planet.id,
+        name = planet.name,
+        description = planet.description,
+        flag = planet.flag,
```

Comment:
I also like the `dict()` version of creating new dictionaries, since the keyword argument style lets me avoid writing the quotes around the keys.

When writing keyword arguments in Python, we usually leave out the spaces around the `=`, so this would be more like
```py
        id=planet.id,
        name=planet.name,
        description=planet.description,
        flag=planet.flag,
```

This may seem strange, but it helps them stand out from regular variable assignments.